profiles/default/linux/amd64/23.0/desktop/gnome/systemd/use.stable.mask

```diff
@@ -1,8 +1,6 @@
 # Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-# Nowa Ammerlaan <nowa@gentoo.org> (2023-12-18)
+# Alfred Wingate <parona@protonmail.com> (2025-09-20)
 # This is still experimental, prebuilt images may not boot on all systems.
-sys-kernel/gentoo-kernel generic-uki
-sys-kernel/gentoo-kernel-bin generic-uki
-sys-kernel/vanilla-kernel generic-uki
+generic-uki
```

Comment:
While we are doing this, let's drop the stable-mask on amd64.

For arm64 we are still waiting on https://bugs.gentoo.org/950118